ambry-api/src/main/java/com.github.ambry/rest/Account.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.rest;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONObject;
+
+
+/**
+ * A representation of an Ambry account. An account is an internal representation of a user.
+ */
+public class Account {
+  private final short id;
+  private final String userId;
+  private final JSONObject metadata;
+  private final List<Container> containers;
+  private final Map<Short, Container> idContainerMap;
+  private final Map<String, Container> nameContainerMap;
+
+  /**
+   * Constructor of Account.
+   * @param id The id of the account.
+   * @param userId The id of the corresponding user.
+   * @param metadata The metadata of the account in JSON.
+   * @param containers The list of containers of the account.
+   */
+  public Account(short id, String userId, JSONObject metadata, List<Container> containers) {
```

Comment:
Also, should we make this `int` or `long` instead of `short`? Storage that wants to optimize can downgrade as required.